Xamarin.Forms.Platform.Android/Platform.cs

```diff
@@ -847,29 +847,30 @@ void SetActionBarTextColor()
 			Color navigationBarTextColor = CurrentNavigationPage == null ? Color.Default : CurrentNavigationPage.BarTextColor;
 			TextView actionBarTitleTextView = null;
 
-			if(Forms.IsLollipopOrNewer)
+			int actionBarTitleId = _context.Resources.GetIdentifier("action_bar_title", "id", "android");
```

Comment:
the only thing I don't like about this change is it changes the preference order of action_bar_title/toolbar usage to be inverse what it was before. It would be good to retain the preference ordering just in case there are any oddball scenarios out there.